num_palindrom.py

```diff
@@ -4,11 +4,14 @@ def highest_value_palindrome(palindrome_string, steps):
         return palindrome_string
     steps-=1
     numbers, count, sol= [], [],[]
```

Comment:
zamiast mieć 3 list możesz użyć słownika
```suggestion
    count = {}
    sol = []
```
gdzie count byłby słownikiem gdzie klucz to liczba z palindromu, a wartość o liczba wystąpień w stringu
w ten sposób skróciłby Ci się trochę kod